libraries/dawg-show/src/main/webapp/views/simplified.jsp

```diff
@@ -158,12 +133,14 @@ String fullAudioUrlOGGExtension = audioUrl + "/play1.ogg";
                 <div id="remote" class="remoteContainer" style="width=90%;">
                     <jsp:include page="<%=stdRemotePage %>" />
                 </div>
+                <% if (!isXR11) { %>
```

Comment:
this mute button is the red button to mute the audio stream from the browser, not sending it to the box itself.
Because we are using XR11 remote, I've merged those 2 buttons together, but for compatibility reasons, I am hiding it only for XR11